test/object-store/sync/flx_sync.cpp

```diff
@@ -167,7 +147,7 @@ TEST_CASE("flx: connect to FLX-enabled app", "[sync][flx][app]") {
 
         wait_for_download(*realm);
         {
-            wait_for_advance(realm);
+            realm->refresh();
```

Comment:
I wasn't sure what `wait_for_advance()` was trying to do and took a guess, as it didn't actually do anything. It would have if `RealmCoordinator::can_advance()` did wait it claimed it did, but that comment wasn't updated when something different was implemented as part of the core 6 port. If the goal is to verify that autorefresh works properly for writes from sync bootstrapping then `refresh()` isn't the correct answer and I'll change it to wait for the appropriate notification.